shared/chat/conversation/messages/message-popup/exploding/container.tsx

```diff
@@ -123,7 +123,7 @@ export default Container.connect(
     _onKick: (teamID: TeamTypes.TeamID, username: string) =>
       dispatch(
         RouteTreeGen.createNavigateAppend({
-          path: [{props: {navToChat: true, teamID, username}, selected: 'teamReallyRemoveMember'}],
+          path: [{props: {members: [username], teamID}, selected: 'teamReallyRemoveMember'}],
```

Comment:
I think there have been issues with extra renders with single-member props arrays in the past, might be good to do something like https://github.com/keybase/client/pull/22986 and accept either an array or a single username

Reply:
Hmm, I think the extra renders would be because the parent keeps remaking the array, but if it's a routeprop like it is here, the array would only get made once, so I claim we shouldn't need to do that here. WDYT?

Reply:
that sounds reasonable to me!